The task_annotator member variable was made in public in
crrev.com/2e146d74f78041f91bd43562bd1a0210374a41eb and the AlarmTimer
class in libchromeos needs this member variable to be public.  This
patch can be dropped when we uprev libchrome.

diff --git a/base/message_loop/message_loop.h b/base/message_loop/message_loop.h
index 8db1e77..69457f7 100644
--- a/base/message_loop/message_loop.h
+++ b/base/message_loop/message_loop.h
@@ -391,6 +391,10 @@ class BASE_EXPORT MessageLoop : public MessagePump::Delegate {
   // Returns true if the message loop is "idle". Provided for testing.
   bool IsIdleForTesting();
 
+  // Returns the TaskAnnotator which is used to add debug information to posted
+  // tasks.
+  debug::TaskAnnotator* task_annotator() { return &task_annotator_; }
+
   //----------------------------------------------------------------------------
  protected:
   scoped_ptr<MessagePump> pump_;
@@ -423,10 +427,6 @@ class BASE_EXPORT MessageLoop : public MessagePump::Delegate {
   // true if some work was done.
   bool DeletePendingTasks();
 
-  // Returns the TaskAnnotator which is used to add debug information to posted
-  // tasks.
-  debug::TaskAnnotator* task_annotator() { return &task_annotator_; }
-
   // Loads tasks from the incoming queue to |work_queue_| if the latter is
   // empty.
   void ReloadWorkQueue();
